source/style-guide/style/links-and-cross-references.txt

```diff
@@ -107,8 +108,8 @@ Cross Reference and Link Formatting
 
   .. note::
 
-     Don't create a TOC creating one manually within the article. The
-     ``.. contents::`` directive generates a TOC if you add it.
+     Don't create a TOC creating manually within the article. Add the
```

Comment:
Delete "manually"

Reply:
Deleted "creating". Don't know how I missed that.